app/views/users/single_market.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1>
+  <% market = @market %>
```

Comment:
This line is unnecessary.  You could just use the instance variable in the below places that you have "market"